server/liveblog/themes/template/filters.py

```diff
@@ -57,3 +57,26 @@ def addten(date):
         return re.sub(r'(\d{4})', regaddten, date)
 
     return date
+
+
+def ampify(html):
+    if re.search('iframe', html, re.IGNORECASE):
+        src = re.search(r'src\s*=\s*"(?P<src>[^\"]+)"', html)
+        width = re.search(r'width\s*=\s*"(?P<width>[^\"]+)"', html)
+        height = re.search(r'height\s*=\s*"(?P<height>[^\"]+)"', html)
+        print(src.group('src'))
```

Comment:
R U sure you want to print anything here?

Reply:
now you see that reviews are good :dagger: